raven/processes/wps_forecast_flood_risk.py

```diff
@@ -99,7 +99,9 @@ def _handler(self, request, response):
             )  # This is needed to return values instead of floats
 
         # Put in file
+
         filename = tempfile.mkdtemp() + "/fcst_flood_risk.nc"
+        pct.name = "exceedance_probability"
```

Comment:
Here you're converting a DataArray to a Dataset, but the Dataset will have no global attributes. 
I suggest doing something like
```
out = pct.to_dataset(name="exceedance_probability")
out.attrs = ...
out.to_netcdf(filename)
```
where you copy attributes from the original file, or create new attributes describing the file content. You probably minimally want attributes like title, source and history.